File: app/src/main/java/com/example/learningverbs/tools/LearningApplication.java
package com.example.learningverbs.tools;

import android.app.Application;
import android.content.Context;

import androidx.appcompat.app.AppCompatDelegate;

public class LearningApplication extends Application {
  private static Context myApplicationContext;

  @Override
  public void onCreate() {
    super.onCreate();
    AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
    myApplicationContext = this;


  }
  public static Context getMyApplicationContext(){
    return myApplicationContext;
  }

}

File: app/src/main/java/com/karla/learningverbs/kotlin/utils/LearningApplication.kt
package com.karla.learningverbs.kotlin.utils

import android.app.Application
import android.content.Context

class LearningApplication: Application() {
    companion object {
        var myApplicationContext: Context? = null
        @JvmField
        var appInstance: LearningApplication? = null

        @JvmStatic fun getInstance(): LearningApplication {
            return appInstance as LearningApplication
        }
    }

    val applicationName: String
        get() = LearningApplication.getInstance().applicationInfo.loadLabel(baseContext.packageManager).toString()

    override fun onCreate() {
        super.onCreate()
        appInstance=this;
    }

}